src/main/java/edu/ucla/library/libcal/services/OAuthTokenService.java

```diff
@@ -0,0 +1,86 @@
+
+package edu.ucla.library.libcal.services;
+
+import edu.ucla.library.libcal.Config;
+import edu.ucla.library.libcal.Constants;
+
+import io.vertx.codegen.annotations.GenIgnore;
+import io.vertx.codegen.annotations.ProxyClose;
+import io.vertx.codegen.annotations.ProxyGen;
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.auth.oauth2.OAuth2Auth;
+import io.vertx.ext.auth.oauth2.OAuth2FlowType;
+import io.vertx.ext.auth.oauth2.OAuth2Options;
+import io.vertx.serviceproxy.ServiceProxyBuilder;
+
+/**
+ * A service for managing the OAuth token used by LibCal API clients.
+ */
+@ProxyGen
+@VertxGen
+public interface OAuthTokenService {
```

Comment:
+1 on the service. I'd thought about suggesting that last time instead of the verticle, but figured we'd punt it to a future ticket. I like it, though.